recipes/libpcsclite/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "libpcsclite" %}
```

```suggestion
{% set name = "pcsclite" %}
```
This package also contains binaries with CLI. libpcsclite would be the subpackage of this recipe with only the library/headers. You can split the package into multiple outputs now if you want, but it's not necessary.

---

Yes indeed.